composer.json

```diff
@@ -1,17 +1,31 @@
 {
-	"name": "openemr/openemr",
-	"description": "OpenEMR is a Free and Open Source electronic health records and medical practice management application",
-	"license": "GPL",
+    "name": "openemr/openemr",
+    "description": "OpenEMR is a Free and Open Source electronic health records and medical practice management application",
+    "license": "GPL",
     "support": {
         "website": "http://www.open-emr.org/",
         "issues": "https://sourceforge.net/p/openemr/bugs/",
         "forum": "https://sourceforge.net/p/openemr/discussion/",
         "wiki": "http://www.open-emr.org/wiki/index.php/OpenEMR_Wiki_Home_Page",
         "source": "http://github.com/openemr/openemr"
     },
-	"require": {
+    "require": {
 
-        "smarty/smarty": "2.6.29"
-		
-	}
+        "smarty/smarty": "2.6.29",
+        "adodb/adodb-php": "5.20.2"
+        
+    },
+    "require-dev": {
```

Comment:
After above issue is cleared up (if it is even an issue), will plan to bring this into the codebase.